dashboard/modules/runtime_env/runtime_env_agent.py

```diff
@@ -37,7 +39,8 @@
 )
 from ray.runtime_env import RuntimeEnv, RuntimeEnvConfig
 
-default_logger = logging.getLogger(__name__)
+RUNTIME_ENV_AGENT_LOGGER_NAME = "runtime_env_agent_logger"
```

Comment:
nit: Could we have this and other logger names extracted to `ray_constants.py`? For logger names that differs from the file/module name? 

Else I'm not sure what the benefit of using a different name than `__name__` would be.